Docs/readme_ja.md

```
@@ -0,0 +1,133 @@
+<div align="center">
+  <h1 align="center">
+    <a href="https://www.bettergi.com/"><img src="https://img.alicdn.com/imgextra/i2/2042484851/O1CN014wn1rf1lhoFYjL0gA_!!2042484851.png" width="200"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)